packages/regl-worldview/src/Worldview.js

```diff
@@ -196,8 +196,12 @@ export class WorldviewBase extends React.Component<BaseProps, State> {
       return;
     }
 
-    if (!readHitmap && worldviewHandler) {
-      return handleWorldviewMouseInteraction(0, ray, e, worldviewHandler);
+    // rendering the hitmap on mouse move is expensive, so disable it by default
+    if (mouseEventName === "onMouseMove" && !this.props.hitmapOnMouseMove) {
```

Comment:
Could you pull out into variable like `const renderHitMapOnMouseMove = mouseEventName === "onMouseMove" && this.props.hitmapOnMouseMove` and say `if (!renderHitmapOnMouseMove)`? Would that do what you want it to do?